test/plugins/oas/index.js

```diff
@@ -63,6 +63,15 @@ export default async ({ specPath, ajvBodyOptions = {}, ajvHeadersOptions = {} })
 
 		chai.Assertion.addMethod('matchApiSchema', function () {
 			let response = this._obj;
+
+			if (typeof response.body === 'string') {
+				response.body = JSON.parse(response.body);
+			}
```

Comment:
Any reason why we're parsing ourselves in both places instead of passing `responseType: 'json'` to `got` ? If yes, let's at least swap this with the if below and add `&& response.type === 'application/json'` here.